packages/lodestar/test/benchmarks/index.ts

```diff
@@ -0,0 +1,12 @@
+import {createReportFile} from "./utils"
+
+// Import benchmarks
+import * as benchmarks from "./imports";
+
+// Create file
+const file = createReportFile();
+
+// Run benchmarks
+for (let bench in benchmarks) {
```

Comment:
I think it might be nice to have benchmarks saved in separate files.

eg:
```
benchmark-reports/
  2019-07-24T19:42:22.081Z/
    function1.txt
    function2.txt
    ...etc
```

Reply:
hmm ya ok thats doable